### PR TITLE
Lesson 05 task

### PR DESCRIPTION
**Pull request by karamel29**

### CODE REVIEW

File: task005/shape/rectangle/rectangle_test.go [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+	"github.com/stretchr/testify/require"
+)
+
+// rectangle area tests
+func TestRectangle_Area(t *testing.T) {
+	type fields struct {
+		height float64
+		width  float64
+	}
+	tests := []struct {
```
Comment (DzmitryYafremenka) [inline task005/shape/rectangle/rectangle_test.go:16]: You should add more edge cases, for example if one of params = math.MaxFloat64.

File: task005/shape/circle/circle_test.go [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+
+	"github.com/stretchr/testify/require"
+)
+
+// circle perimeter tests
+func TestCircle_Perimeter(t *testing.T) {
+	type fields struct {
+		radius float64
+	}
+	tests := []struct {
```
Comment (DzmitryYafremenka) [inline task005/shape/circle/circle_test.go:16]: if i pass math.MaxFloat64 as a radius - this code will pass all of the tests, and validations.

File: task005/shape/circle/circle.go [hunk 3]
```diff
@@ -0,0 +1,36 @@
+package circle
```
Comment (IgorZhadenovEpam) [inline task005/shape/circle/circle.go:1]: It looks like you have not quite figured out the interfaces in GO. 
You have to implement Stringer interface. https://pkg.go.dev/fmt#Stringer
Any type that has a `String () string` method implements the Stringer interface. https://tour.golang.org/methods/17
So you have to add 
```
func (c Circle) String() string{
...
}

```
and test it accordingly.
The same about Rectangle.
